### PR TITLE
1909/story 7658 internal server 500 errors should include timestamp in the toast message

### PR DESCRIPTION
**Pull request by collin-mcgrath**

### CODE REVIEW

File: test/unit/exception_handling_test.rb [hunk 1]
```diff
@@ -111,10 +111,11 @@ class SmtpClientErrbackStub < SmtpClientStub
 
     context "#log_error" do
       should "take in additional keyword args as logging context and pass them to the logger" do
-        ExceptionHandling.log_error('This is an Error', 'This is the prefix context', service_name: 'exception_handling')
+        error_id = ExceptionHandling.log_error('This is an Error', 'This is the prefix context', service_name: 'exception_handling')
```
Comment (jebentier) [inline test/unit/exception_handling_test.rb]: instead of shoehorning this into and existing test, could we create a more targeted test for the new contract? repurposing the deleted `should "return nil"` test seems like a good thing to do.

Comment (collin-mcgrath) [reply test/unit/exception_handling_test.rb]: I thought the assertion would speak for itself. I tend to think we sometimes overdo it by breaking out so many individual assertions into their own blocks. Repurposing the existing block seems fine here though and I don't feel too strongly about it. 

Done.

Comment (jebentier) [reply test/unit/exception_handling_test.rb]: This is definitely something to keep in mind, but i think if tests are slim and direct and testing single responsibility, the cognitive load of figuring out what broke given a change that was made is much more obvious.  The `should` statements should give very direct statements of what is expected to happen so that they point directly to the issue, rather than relying on the assertion of `expected x to be y` to indicate that you broke a return somewhere.
